### PR TITLE
fix(kit): `CalendarRange` shows actual `defaultViewedMonth`

### LINKED ISSUE #8494: 🐞 - `CalendarRange` updating `defaultViewedMonth` not working
### Playground Link

https://taiga-ui.dev/v3/components/calendar-range/API?defaultViewedMonth$=2

### Description

1. Open https://taiga-ui.dev/v3/components/calendar-range/API
2. Select other `defaultViewedMonth`

https://github.com/user-attachments/assets/f6f4e8d1-c419-429f-90b6-cb1aa074fae4

### Angular version

12

### Taiga UI version

3.89.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by mdlufy**
Close #8494 

**As Is:**

https://github.com/user-attachments/assets/9d1798af-8125-4f7d-98d3-c2bc93e92c5f

**To Be:**

https://github.com/user-attachments/assets/65c967e3-45f9-4c65-8853-0c7645e4c178

### CODE REVIEW

**Review by splincode — APPROVED** [review]

**Review by splincode — APPROVED** [review]

File: projects/kit/internal/primitive-calendar-range/primitive-calendar-range.component.ts [hunk 1]
```diff
@@ -99,6 +101,16 @@ export class TuiPrimitiveCalendarRangeComponent implements OnInit {
     monthOffset: TuiTypedMapper<[TuiMonth, number], TuiMonth> = (value, offset) =>
         value.append({month: offset});
 
+    ngOnChanges({
+        defaultViewedMonthFirst,
+        defaultViewedMonthSecond,
+    }: SimpleChanges): void {
+        this.updateViewedMonths(
+            defaultViewedMonthFirst?.currentValue,
```
Comment (waterplea) [inline projects/kit/internal/primitive-calendar-range/primitive-calendar-range.component.ts:109]: If you only need current value you can just use the values as is, no need for `SimpleChanges`

Comment (mdlufy) [reply projects/kit/internal/primitive-calendar-range/primitive-calendar-range.component.ts:109]: No, we need to check if it comes new value of `defaultViewedMonth`

**Review by waterplea — APPROVED** [review]

**Review by mdlufy — COMMENTED** [review]
WIP: need to fix case when value in right calendar selected - it's switched no to the next month

**Review by mdlufy — COMMENTED** [review]
WIP: need to fix case when value in right calendar selected - it's switched no to the next month

**Comment by mdlufy** [comment]
> WIP: need to fix case when value in right calendar selected - it's switched no to the next month
Fixed

**Pull request merged**